certificate_path.go

```diff
@@ -0,0 +1,6 @@
+package globalsign
```

Comment:
https://github.com/unidoc/unipdf/wiki/UniPDF-Developer-Guide#8-license-header-in-each-go-file
Need to add a license too... Probably we can distribute this as MIT, but maybe need to make a note that it links to UniPDF which has a higher license requirement, i.e. EULA?

Reply:
Updated at 1e197714ba820fb2d596cd1880cdf67ff93abed0